cmd/contour/serve.go

```diff
@@ -231,6 +231,11 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 	// register observer for endpoints updates.
 	endpointHandler.Observer = contour.ComposeObservers(snapshotHandler)
 
+	dnsLookupFamily, err := ParseDNSLookupFamily(ctx.DNSLookupFamily)
+	if err != nil {
+		log.WithField("context", "dns-lookup-family").Infof("error parsing config parameter dns-lookup-config %q", ctx.DNSLookupFamily)
```

Comment:
in other places within this function we return parse errors (e.g. timeouts, default HTTP versions) rather than just logging them - I think we should probably do the same here for consistency.